raiden/transfer/mediated_transfer/target.py

```diff
@@ -217,7 +219,7 @@ def handle_onchain_secretreveal(
             secret_reveal_block_number=state_change.block_number,
         )
 
-        target_state.state = 'reveal_secret'
+        target_state.state = 'onchain_unlock'
```

Comment:
see above: by not using constants, we open ourselves up for simple typos that lead to unexpected behavior; especially so in rarely used code paths.

Also: `'onchain_unlock'` is not in raiden/transfer/mediated_transfer/state.py `TargetTransferState.valid_states`, as far as I can tell.